src/TimeModeling/time_modeling_serial.jl

```diff
@@ -36,24 +36,42 @@ function time_modeling(model_full::Model, srcGeometry::Geometry, srcData, recGeo
 
     # Extrapolate input data to computational grid
     if mode==1  # F and J
+        if dtComp==srcGeometry.dt[1]
+        qIn = srcData[1];
+        else
         qIn = time_resample(srcData[1],srcGeometry,dtComp)[1]
+        end
```

Comment:
So from what I see this is the same if/else everywhere. So i would leave this function as it is and add the if/else inside the `time_resample` resample instead.

Reply:
ok, I will change the time-resample and keep this clean

Reply:
Thanks

Reply:
@mloubout  when u say odd things, you mean its not indenting the loops properly?